defaults/python/lib/plugin/settings.py

```diff
@@ -83,6 +85,7 @@ class HostSettings(TypedDict):
     audio: AudioSettings
     runnerTimeouts: RunnerTimeouts
     passToMoonlight: bool
+    showPerformanceStats : bool
```

Comment:
```suggestion
    showPerformanceStats : Optional[bool]
```
We also need a default value for not doing anything.

Reply:
Done